aquadoggo/src/replication/target_set.rs

```diff
@@ -52,13 +66,45 @@ impl Validate for TargetSet {
         };
 
         let mut prev_schema_id: Option<&SchemaId> = None;
+        let mut initial_system_schema = true;
+
+        // We need to validate that:
+        // - if system schema are included they are first in the list and ordered alphabetically
+        // - any following application schema are also ordered alphabetically
+        for (index, schema_id) in self.0.iter().enumerate() {
```

Comment:
Do we need any tests checking that new validation logic?

Reply:
We sure do: https://github.com/p2panda/aquadoggo/pull/387/commits/c5373eff837ff15c36542480fe418eaa63b8913d

Reply:
Amazing!